.npmignore

```diff
@@ -1,4 +1,6 @@
 .firebaserc
 .github
+firebase.json
+postcss.config.cjs
```

Comment:
makes sense to omit firebase config, but i think leaving the postcss config makes sense in case someone npm installs this and wants to build it themselves?

Reply:
IMO, in this case, you can use the source code, right?
Is a strange use case to build a package in side the `node_modules` folder

Any way, I was remove the `postcss.config.cjs`